src/pages/SubmissionHistory.js

```diff
@@ -0,0 +1,215 @@
+import _ from 'lodash/fp'
+import { Fragment, useState } from 'react'
+import { div, h, h2 } from 'react-hyperscript-helpers'
+import { AutoSizer } from 'react-virtualized'
+import { ButtonOutline, headerBar, Link } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import { makeStatusLine, statusType } from 'src/components/submission-common'
+import { FlexTable, paginator, Sortable, tableHeight, TextCell } from 'src/components/table'
+import { Ajax } from 'src/libs/ajax'
+import * as Nav from 'src/libs/nav'
+import { notify } from 'src/libs/notifications'
+import { useCancellation, useOnMount } from 'src/libs/react-utils'
+import * as Utils from 'src/libs/utils'
+
+
+export const SubmissionHistory = () => {
+  // State
+  const [sort, setSort] = useState({ field: 'submission_timestamp', direction: 'desc' })
+  const [pageNumber, setPageNumber] = useState(1)
+  const [itemsPerPage, setItemsPerPage] = useState(50)
+  const [runSetsData, setRunSetData] = useState()
+
+  const signal = useCancellation()
+
+  const LOREM_IPSUM = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
```

Comment:
Is this still needed? Is this functionality to be added after we add the submission pages?

Reply:
Not sure when we're planning to add comment support to the API, but I think it's not implemented yet